src/app/core/components/header/header.component.html

```diff
@@ -8,7 +8,7 @@
       <mat-icon>perm_identity</mat-icon>
       User login
     </button>
-    <button *appIfAuthenticated="this.authService.isLoggedIn">
+    <button *appIfAuthenticated="isLoggedIn">
```

Comment:
The button doesn't appear on the page. Use getter to get auth info.
Click on the button should logout user and redirect to the login page. Right now there is no event handler on this button, so it doesn't work.

Reply:
The logout button still does not work. Click on the button should log the user out (remove the token from the local storage) and redirect to the login page.